tests/system/large/functions/test_remote_function.py

```diff
@@ -415,8 +429,10 @@ def test_remote_function_explicit_dataset_not_created(
     try:
 
         @session.remote_function(
-            input_types=[int],
-            output_type=int,
+            # Make sure that the input/output types can be used positionally.
+            # This avoids the worst of the breaking change from 1.x to 2.x.
+            [int],
+            int,
             dataset=dataset_id_not_created,
```

Comment:
I missed this one, but I think it's good to have some tests with the keyword version of the parameter too.